sdks/python/apache_beam/utils/pipeline_options_validator_test.py

```diff
@@ -87,7 +87,7 @@ def get_validator(temp_location):
       return validator
 
     test_cases = [
-        {'temp_location': None, 'errors': ['temp_location']},
```

Comment:
This dependency doesn't exist in pipline_options_validator though, is there another test suit where I can add such a test?